go.mod

```diff
@@ -285,7 +285,8 @@ require (
 )
 
 // Using a fork of Prometheus with Mimir-specific changes.
-replace github.com/prometheus/prometheus => github.com/grafana/mimir-prometheus v0.0.0-20241210170917-0a0a41616520
+// TODO: THIS IS A BRANCH, REVERT TO MAIN BEFORE MERGING THIS PR
+replace github.com/prometheus/prometheus => github.com/grafana/mimir-prometheus v0.0.0-20241213165041-26afe3d921cd
```

Comment:
Please remember to update mimir prometheus before merging this PR - update it in a separate PR.